src/screens/MyWorkouts/WorkoutSearchBar.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import { ScrollView, TextInput } from 'react-native-gesture-handler';
+import { supabase } from '../../utils/supabaseClient';
+
+/**
+ * @param selectedExercise.exercise //value set to exercise pressed
+ * @param selectedExercise.isOk //if true, then value of exercise is valid, otherwise disregard
```

Comment:
I think since we will have full end-to-end control of the exercise values, this `isOk` prop is probably redundant and we can get rid of it.

Reply:
I only put that because what if you click add exercise but want to cancel it, selectedExercise.excercise value might be unknown